src/course-outline/card-header/CardHeader.jsx

```diff
@@ -50,16 +54,18 @@ const CardHeader = ({
   discussionEnabled,
   discussionsSettings,
   parentInfo,
-  tagsCount,
 }) => {
   const intl = useIntl();
   const [searchParams] = useSearchParams();
   const [titleValue, setTitleValue] = useState(title);
   const cardHeaderRef = useRef(null);
+  const [isManageTagsDrawerOpen, openManageTagsDrawer, closeManageTagsDrawer] = useToggle(false);
 
   const isDisabledPublish = (status === ITEM_BADGE_STATUS.live
     || status === ITEM_BADGE_STATUS.publishedNotLive) && !hasChanges;
 
+  const { data: contentTagCount, isSuccess: isContentTagCountSuccess } = useContentTagsCount(cardId);
+
```

Comment:
Each item asks for its tag count. As commented above, we could prefetch it (to prevent multiple API calls), but I think we don't need it.